docs/blog/v5-getting-started.md

```diff
@@ -176,7 +176,7 @@ Yes we did, so in the next session lets use it.
 
 ### Ingress example
 
-Now lets use the ingress example instead, you can find the [example](../../examples/ingress_http/resources.yaml).
+Now lets use the ingress example instead, you can find the [example]({{< ref "docs/examples/ingress_http/readme/" >}}).
```

Comment:
Are examples pulled in under the `docs/` directory or should there be a `ref` to the `/examples/` dir?

Reply:
It's just easier to point at a section instead of nested relative refs. `/docs/examples` is a correct, you can check it the already deployed docs: https://grafana.github.io/grafana-operator/docs/examples/